ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/visitor/infraestructure/antlr/visitors/classes/JavaConstructorVisitor.java

```diff
@@ -0,0 +1,59 @@
+package edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.classes;
+
+import java.util.ArrayList;
+import java.util.List;
+import edu.usb.argos.ASTProcessor.antlr.JavaParser;
+import edu.usb.argos.ASTProcessor.antlr.JavaParserBaseVisitor;
+import edu.usb.argos.ASTProcessor.visitor.core.entities.classes.ConstructorInfo;
+import edu.usb.argos.ASTProcessor.visitor.core.interfaces.visitor.IConstructorAnalyzer;
+
+public class JavaConstructorVisitor extends JavaParserBaseVisitor<List<ConstructorInfo<JavaParser.BlockStatementContext>>> implements IConstructorAnalyzer<JavaParser.ClassBodyContext, JavaParser.BlockStatementContext> {
+
+    @Override
+    public List<ConstructorInfo<JavaParser.BlockStatementContext>> visitConstructors(JavaParser.ClassBodyContext ctx) {
+        List<ConstructorInfo<JavaParser.BlockStatementContext>> constructors = new ArrayList<>();
+
+        for (JavaParser.ClassBodyDeclarationContext bodyCtx : ctx.classBodyDeclaration()) {
+            JavaParser.MemberDeclarationContext memberCtx = bodyCtx.memberDeclaration();
+
+            if (memberCtx != null && memberCtx.constructorDeclaration() != null) {
+                JavaParser.ConstructorDeclarationContext constructorCtx = memberCtx.constructorDeclaration();
+                String constructorName = constructorCtx.identifier().getText();
+                List<String> modifiers = getModifiers(bodyCtx);
+                List<String> parameters = getParameters(constructorCtx);
+                List<JavaParser.BlockStatementContext> bodyStatements = getBodyStatements(constructorCtx);
+
+                constructors.add(new ConstructorInfo<>(constructorName, modifiers, parameters, bodyStatements));
```

Comment:
Don't you think if it'd be good to create the object before instead of creating that moment? At the final, we return a list of `BlockStatementContext`.

Reply:
The statements are made up of tokens and it is better that this manipulation is done by the concretes of the interfaces as generic

Reply:
I see you finally created an object furthermore of using builder pattern, so it's good.